blog/2020-10-12-ambrosia-alpha-release.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Trade Bitcoin and DAI with COMIT"
+author: franck
+author_url: https://github.com/d4nte
+author_image_url: https://avatars1.githubusercontent.com/u/300805
+tags: [bitcoin,dai,atomic swap,ambrosia]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# A Change of Direction
+
+Last year, we spent time and effort creating tools, such as [`create-comit-app` and `comit-scripts`](https://github.com/comit-network/create-comit-app/) to allow developers to build their own COMIT Apps.
+We organised local meetups in Sydney and even a hackathon in Singapore.
+We had a number of developers excited about the technology and possibilities that COMIT bring in but very few committed to build an app.
+
+We now realized that before we can expect others build successful apps with our tools, we should demonstrate what can be done with COMIT ourselves.
+
+<!--truncate-->
+
+# Introducing Ambrosia
+
+<img alt="Ambrosia" src={useBaseUrl('blog/assets/images/2020-10/ambrosia.png')} />
+
+This is why we spent the last few months working on [ambrosia](https://github.com/comit-network/ambrosia/), an UI for trading in the COMIT network.
+Ambrosia allows you to trade BTC/DAI in a decentralised manner. No third party to trust, no central orderbook, no KYC.
+It uses [cnd](https://github.com/comit-network/ambrosia/comit-rs) as backend.
+
+The software has some rough edges, but we wanted to get it out there to have your feedback.
+Here are some choices we want your input on:
+
+## Custody
+
+Not only do you not need to trust a third party with your funds when trading, even our software is completely non-custodial.
+Ambrosia never holds your crypto, instead it interfaces directly with your hardware wallet (nano ledger s only) to make transactions.
```

Comment:
I propose to rephrase this as I had to read the first sentence twice to understand it: 
```suggestion
Ambrosia never holds your crypto, instead it interfaces directly with your hardware wallet (nano ledger s only) to make transactions. 
This is a huge step up towards trustless trading compared to traditional centralized solutions where you have to deposit your funds into an exchange first. 
```